components/Post.vue

```diff
@@ -45,7 +45,21 @@
 <script>
 export default {
 	// Pages have no media attached to them so when they render they are imageless
-	props: ["picture", "title", "date", "slug"]
+	props: {
+		picture: {
+			type: Object,
+			default: () => "https://wecount.inclusivedesign.ca/wp-content/uploads/2019/10/We-Count-logos_colour-and-bw-01.png"
+		},
+		title: {
+			type: String,
+			default: "We Count Default"
+		},
+		date: Date,
+		slug: {
+			type: String,
+			default: "/"
```

Comment:
Is it possible to omit the default?

Reply:
Testing reply

Reply:
Not providing the defaults trigger `vue/require-default-prop` WARNING messages. An explanation of this rule can be found [here](https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/require-default-prop.md)